src/github.ts

```diff
@@ -15,6 +16,7 @@ export const getContext = (): Context => {
   return {
     repo: getRepo(),
     eventName: getEnv('GITHUB_EVENT_NAME'),
+    refName: getEnv('GITHUB_REF_NAME'),
```

Comment:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables